react/daily-art-prompt/src/components/CommentLayout.js

```diff
@@ -2,14 +2,48 @@ import React from 'react';
 
 import Comment from './Comment.js'
 
-const CommentLayout = ({ comment, onClick }) =>
-    <div>
-        <Comment
-            comment={comment.text}
-        />
-        <button onClick={onClick}>
-            Delete
-        </button>
-    </div >
+const CommentLayout = ({
+    comment,
+    onDelete,
+    onCancel,
+    onEdit,
+    onSubmit
+}) => {
+
+    return (
```

Comment:
You could return the Jsk implicitly (aka: `=> ( <div>.....`

Reply:
Will do